conference/react-js/src/components/SlideBlock/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SlideBlock = ({ isOpened, children }) => (
+  <div>
+    {
+      isOpened ?
```

Comment:
It is not necessary, but how about ```children[+isOpened]``` ? :)

Reply:
I think it's a good idea.